tools/txs/Cargo.toml

```diff
@@ -34,3 +34,5 @@ url = { workspace = true }
 diem-temppath = { workspace = true }
 libra-framework = { workspace = true }
 libra-smoke-tests = { workspace = true }
+smoke-test = { workspace = true }
+diem-forge = { workspace = true }
```

Comment:
Add trailing newline?

Reply:
Sire I beg forgiveness. It was the wrong Ubuntu version

<img width="586" alt="Screenshot 2024-02-29 at 8 31 21 PM" src="https://github.com/0LNetworkCommunity/libra-framework/assets/1364012/4ff8f373-31cf-408f-84cd-b9a242f86931">